tests/phpunit/modules/test-time-shortcode.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+class TimeShortcodeTest extends WP_UnitTestCase {
+
+	/**
+	* Time Parser tests
+	*/
+
+	function test_parse_time_now() {
+
+		$time_string = 'now';
+		$epoch_now = date( 'U' );
+
+		$parsed_time = o2_Time_Shortcode::parse_time( $time_string );
```

Comment:
It looks like there's a chance for this test to fail (if the second changes between the `date()` call here and the `strtotime()` call in `::parse_time()`. It probably doesn't matter when humans run the tests (a rerun almost certainly won't trigger it), but robots might get mad. Thoughts?